docs/docs/user-guide/using-cli-context.md

```diff
@@ -0,0 +1,107 @@
+# Use CLI for Microsoft 365 context
+
+CLI for Microsoft 365 context may store any kind of option and its value. 
```

Comment:
Could you expand a bit on what a context is and why it is useful, before going into how it works?

Reply:
ok I added more details